src/main/webapp/mojo-dashboard-card-iframe.jsp

```diff
@@ -0,0 +1,19 @@
+<div>
+	<script src="https://code.jquery.com/jquery-3.5.1.min.js" type="text/javascript" unsafe.integrity="sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0=" unsafe.crossorigin="anonymous">
+
+
+	</script>
+
+	<p>
+		<iframe id="frame" src="https://community-ninja-board-ninja-board.int.open.paas.redhat.com/community-ninja-board/mojo/mojo-dashboard-card.jsp" style="width: 100%; height:500px;" border="0"></iframe>
```

Comment:
Should this be a relative path instead of absolute since its hardcoded to prod?

Reply:
no, this is just a code copy/paste for the deploying in an html widget in Igloo. If it was relative it would try loading from the Igloo domain.